sources/packages/api/src/route-controllers/designation-agreement/models/designation-agreement.model.ts

```diff
@@ -35,7 +38,7 @@ export interface LocationsDesignationsDto {
   designationLocationId?: number;
   locationId: number;
   locationName: string;
-  locationData: any;
+  locationData: InstitutionLocationInfo;
```

Comment:
Maybe not part of the PR but maybe would be good to have the `InstitutionLocationInfo` renamed to `InstitutionLocationData` to be aligned with the column name. Just a suggestion.